tests/t252_filter_H.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+from runtest import TestBase
+
+class TestCase(TestBase):
+    def __init__(self):
+        TestBase.__init__(self, 'abc', """
+# DURATION     TID     FUNCTION
+            [ 30217] | main() {
+            [ 30217] |   a() {
+            [ 30217] |     b() {
+   2.500 us [ 30217] |       getpid();
+   8.600 us [ 30217] |     } /* b */
+  11.500 us [ 30217] |   } /* a */
+  14.500 us [ 30217] | } /* main */
+""")
+
+    def prepare(self):
+        self.subcmd = 'record'
+        return self.runcmd()
+
+    def setup(self):
+        self.subcmd = 'replay'
+        self.option = '-H c'
```

Comment:
It should work for 'live' command too.  Then we can just set self.option in the setup function and remove the preprare function.